utils_test.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	dto "github.com/prometheus/client_model/go"
+)
+
+// Check if expected results are in the input, return error if not
+func probeLookup(expRes map[string]float64, mfs []*dto.MetricFamily) error {
```

Comment:
This function doesn't do anything with probes, checkRegistryResults or similar would be better.